Week2/Homework/02_evenandoddreport.js

```diff
@@ -0,0 +1,27 @@
+//Report whether or not a number is odd/even!
+
+//Create a for loop, that iterates from 0 to 20.
+//Create a conditional statement that checks if the value of the counter variable is odd or even.
+//If it's odd, log to the console The number [PUT_NUMBER_HERE] is odd!.
+//If it's even, log to the console The number [PUT_NUMBER_HERE] is even!.
+/*for (var i = 0; i <= 20; i++);
+{
+  if (i === 0) {
+    console.log(i + "is even");
+  } else if (i % 2 === 0) {
+    console.log(i + "is even");
+  } else {
+    console.log(i + "is odd");
+  }
+}*/
+"use strict";
+
+for (let i = 0; i <= 20; i++) {
+  if (i === 0) {
```

Comment:
Ah. I think 0 is considered as an even number. Thus, you need not have a separate condition for logging 0 and could combine it with `i % 2 === 0` condition.